src/InsertManyResult.php

```diff
@@ -35,8 +35,7 @@ class InsertManyResult
     private $isAcknowledged;
 
     /**
-     * @param WriteResult $writeResult
-     * @param mixed[]     $insertedIds
+     * @param mixed[] $insertedIds
```

Comment:
Same comment as I left in BulkWriteResult if you'd like to remove this line, too.